auxiliary/crash-handler/commandline_test.cpp

```diff
@@ -0,0 +1,41 @@
+#include "commandline.hpp"
```

Comment:
Move tests to their own directory, or integrate this into our already-existing tests project